ci/rpm_build_success.sh

```diff
@@ -0,0 +1,20 @@
+# /bin/sh
+
+# Script to be run on successful RPM build
+
+mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+ci_envs="$mydir/ci/parse_ci_envs.sh"
+if [ -e ${ci_envs}"" ]; 
```

Comment:
(style)  trailing whitespace
(lint) Did you forget the 'then' for this 'if'? [SC1049]
(lint) Couldn't parse this if expression. [SC1073]